modules/dispatch/src/lib.rs

```diff
@@ -261,8 +271,11 @@ impl<T: Config<I>, I: 'static> MessageDispatch<T::AccountId, T::BridgeMessageId>
 			},
 		};
 
+		// generate dispatch origin from origin account
+		let origin = T::IntoDispatchOrigin::into_dispatch_origin(&origin_account, &call);
```

Comment:
```suggestion
		let dispatch_origin = T::IntoDispatchOrigin::into_dispatch_origin(&origin_derived_account, &call);
```

Reply:
Check for https://github.com/darwinia-network/darwinia-messages-substrate/pull/121/commits/7e994e2297b533e92b9b1e881c9ae94f4c3edfe7